src/style/mixins.scss

```diff
@@ -22,3 +22,17 @@
       transform: translateY(0.08rem);
     }
 }
+
+@mixin is-flat-clickable() {
+    transition: background-color 0.2s ease, box-shadow 0.2s ease, transform 0.1s ease-out;
+
+    &:hover {
+      box-shadow: var(--button-shadow-hovered);
+    }
+
+    &:active {
+      box-shadow: var(--button-shadow-pressed);
+
```

Comment:
I sometimes put blank lines between properties to group them together. Seems to me like Kia is doing something similar? I'm not sure if the blank line is really necessary when there's just two properties in the block though? 🙂

Reply:
based of Kia's explanation to me , you both share the same thought process on the blank lines.  it might not be necessary but i guess we can allow it on the bases of consistency in grouping